src/test/java/com/znsio/teswiz/screen/android/vodqa/VodqaScreenAndroid.java

```diff
@@ -218,4 +221,57 @@ public boolean isDoubleTapSuccessful() {
         visually.check(SCREEN_NAME,"Double Tap Successful Message", Target.region(byDoubleTapSuccessfulXpath));
         return driver.isElementPresent(byDoubleTapSuccessfulXpath);
     }
+
+    @Override
+    public VodqaScreen navigateToPhotoView() {
+        LOGGER.info("Navigate to Photo View Section");
+        driver.waitTillElementIsVisible(byDoubleTapSuccessfulXpath);
+        driver.waitTillElementIsVisible(byPhotoViewElementXpath).click();
```

Comment:
If both byDoubleTapSuccessfulXpath and byPhotoViewElementXpath are present on the screen,
you can remove check for byDoubleTapSuccessfulXpath By element and keep check for byPhotoViewElementXpath only.

Reply:
byDoubleTapSuccessfulXpath visibility check is a mistake from my side, remove this